server/index.js

```diff
@@ -25,3 +48,6 @@ app.get("/users", (req, res) => {
     }
   });
 });
+
+// All endpoints within this API will be found under the /auth subdirectory
+app.use('/auth', authRouter);
```

Comment:
I don't think the /auth subdirectory is necessary here. I think a user visiting <base_url>/login is a bit more intuitive than <base_url>/auth/login

Reply:
Will do! Also just a note that these are backend APIs so the user will not be accessing these directly